### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by cs-harsh-sharma**
Descriptions:
PMDB Link: https://pmdb.fortinet.com/ProjectManagement/editprojects.php?id=24719

UTCs:
Connector installation verified.
Connector logo verified.
Actions and Playbook list verified.
Playbook tags verified.
All Playbook are in info mode verified.
All Playbook are in inactive mode verified.

Verified all actions.

### CODE REVIEW

File: armis/info.json [hunk 1]
```diff
@@ -549,17 +584,385 @@
...
+          "required": true,
+          "editable": true,
+          "visible": true,
+          "value": "{\n  \"CATEGORY\": \"HANDHELD\",\n  \"MODEL\": \"iPhone 6S\",\n  \"NAME\": \"Mike's iPhone\",\n  \"OS\": \"iOS\",\n  \"OS_VERSION\": \"10.3.1\",\n  \"TYPE\": \"MOBILE_PHONE\"\n}"
+        }
+      ]
+    },
+    {
+      "operation": "add_device_tags",
+      "title": "Add Tags to a Device",
```
Comment (cs-prerna-joshi) [inline armis/info.json]:
```suggestion
      "title": "Add Device Tag",
```

File: armis/info.json [hunk 2]
```diff
@@ -549,17 +584,385 @@
...
+          "editable": true,
+          "visible": true,
+          "placeholder": "e.g. Tag1, Tag2",
+          "value": ""
+        }
+      ]
+    },
+    {
+      "operation": "remove_device_tags",
+      "title": "Remove Tags from a Device",
```
Comment (cs-prerna-joshi) [inline armis/info.json]:
```suggestion
      "title": "Remove Device Tags",
```

File: armis/operations.py [hunk 3]
```diff
@@ -178,49 +256,77 @@ def remove_device_tags(config, params):
...
+
+def _check_health(config):
+    try:
+        arm = Armis(config)
+        if arm.token is not None:
+            return True
+        else:
+            return False
+    except Exception as e:
+        return False
```
Comment (cs-prerna-joshi) [inline armis/operations.py]: should raise the connector error.

File: armis/info.json (hunk 1 above)
Comment (cs-harsh-sharma) [reply armis/info.json]: done

File: armis/info.json (hunk 2 above)
Comment (cs-harsh-sharma) [reply armis/info.json]: done

File: armis/operations.py (hunk 3 above)
Comment (cs-harsh-sharma) [reply armis/operations.py]: done

**Review by cs-prerna-joshi — APPROVED** [review]

**Pull request merged**